tools/nni_cmd/nnictl_utils.py

```diff
@@ -31,24 +31,49 @@
      EXPERIMENT_MONITOR_INFO, TRIAL_MONITOR_HEAD, TRIAL_MONITOR_CONTENT, TRIAL_MONITOR_TAIL
 from .common_utils import print_normal, print_error, print_warning, detect_process
 
-def update_experiment_status():
+def get_experiment_time(port):
+    '''get the startTime and endTime of an experiment'''
+    response = rest_get(experiment_url(port), 20)
```

Comment:
shall we use the default value of the parameter, instead of using "20"?

Reply:
Fixed, changed to use `REST_TIME_OUT`.